ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -242,6 +242,8 @@ public void clearAllCanceledJobs() throws IOException, SignatureException, XmlPr
         }
     }
 
+    /**
+     * Disabling until the TruncateJob request is implemented.
```

Comment:
i think that is better to use the @Ignore, that way you can see that this test is ignored